spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -1815,17 +1815,17 @@ public static class Admin {
 		private Set<String> tlsProtocols;
 
 		/**
-		 * Connection time out.
+		 * Duration to wait for a connection to server to be established.
 		 */
-		private Duration connectTimeout = Duration.ofMinutes(1);
+		private Duration connectionTimeout = Duration.ofMinutes(1);
```

Comment:
Although it is called `connectTimeout` internally - that is a factor of it being used for the connectTimeout on the jax-rs client and is the same "connection timeout". I think keeping it named the same outwardly makes it less confusing.